autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/imputation/SimpleImputer.py

```diff
@@ -15,92 +13,154 @@
 
 
 class SimpleImputer(BaseImputer):
+    """An imputer for categorical and numerical columns
+
+    Impute missing values for categorical columns with 'constant_!missing!'
+
+    Note:
+        In case of numpy data, the constant value is set to -1, under the assumption
+        that categorical data is fit with an Ordinal Scaler.
     """
-    Impute missing values for categorical columns with '!missing!'
-    (In case of numpy data, the constant value is set to -1, under
-    the assumption that categorical data is fit with an Ordinal Scaler)
-    """
 
-    def __init__(self,
-                 random_state: Optional[Union[np.random.RandomState, int]] = None,
-                 numerical_strategy: str = 'mean',
-                 categorical_strategy: str = 'most_frequent'):
+    def __init__(
+        self,
+        random_state: Optional[Union[np.random.RandomState, int]] = None,
+        numerical_strategy: str = 'mean',
+        categorical_strategy: str = 'most_frequent'
+    ):
+        """
+        Parameters
```

Comment:
Actually, we follow google style docstring. You can find more information about it [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). Also, you can check out the docstrings as they are used in AutoPyTorch [here](https://github.com/automl/Auto-PyTorch/blob/ef8d21ae982883548a8e77a3690eefa810d3ae07/autoPyTorch/api/base_task.py#L107).